WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -214,12 +307,32 @@ private extension TotalsView {
         .cornerRadius(Constants.newOrderButtonCornerRadius)
     }
 
+    private var receiptButton: some View {
+        Button(action: {
```

Comment:
Comes back to my question here pdfdoF-5RR-p2#comment-6896

Whether the presentation of these buttons should be controlled through CollectPaymentUseCase and alert providers. Then POS would know what kind of options are available and can choose to show those options and call necessary callbacks.